test/dotnet-add-p2p.Tests/dotnet-add-p2p.Tests.csproj

```diff
@@ -20,6 +20,12 @@
     <ProjectReference Include="..\..\src\Microsoft.DotNet.Cli.Utils\Microsoft.DotNet.Cli.Utils.csproj">
       <FromP2P>true</FromP2P>
     </ProjectReference>
+    <ProjectReference Include="..\..\src\Microsoft.DotNet.Cli.Sln.Internal\Microsoft.DotNet.Cli.Sln.Internal.csproj">
+      <FromP2P>true</FromP2P>
```

Comment:
@livarcocc what does the FromP2P do in this context? Why do we do that everywhere else? i just followed the pattern i've seen in this file